addons/core/XEH_PREP.sqf

```diff
@@ -177,6 +179,8 @@ PREP_SUB(hint,showRadioVolume);
 PREP(setChannelFrequency);
 PREP(setRadioOwner);
 PREP(setLrSpeakers);
+PREP(setSwHalfDuplexOverride);
+PREP(setLrHalfDuplexOverride);
```

Comment:
Sounds like a good idea, I'll merge both functions and rename it to `setHalfDuplexOverride`/`getHalfDuplexOverride`. Will just have to add a few checks for array/string to distinguish, but shouldn't be a problem since the functions don't actually include much logic anyways.